### ISSUE TITLE
Wrong image path 

### ISSUE THREAD

**Issue by daya0576** [zhuzhuyule/HexoEditor#44]
My HexoEditor Setting → Image Source Center: `.../source/images`

When pasting img to HexoEditor, it converted image to`![](/2018-08-07-octopress-to-hexo/old_blog.png)`.

---

However, I expected it to be `![](/images/2018-08-07-octopress-to-hexo/old_blog.png)`

How can I achieve that?

**Comment by zhuzhuyule** [comment]
no, it does not support. the path relative base path( image source center), so it doesn't support!

**Comment by me10zyl** [comment]
I have the same issue, can we configure a custom image prefix?
